test/core/end2end/h2_ssl_cert_test.cc

```diff
@@ -37,6 +39,8 @@
 
 #include <gtest/gtest.h>
 
+static std::string test_server1_key_id;
```

Comment:
nit: use grpc::string is preferred. 
You can remove line 23 as well.

Reply:
Changed.